src/libopensc/card-muscle.c

```diff
@@ -399,6 +390,8 @@ static int select_item(sc_card_t *card, const sc_path_t *path_in, sc_file_t ** f
 		file->path = *path_in;
 		file->size = file_data->size;
 		file->id = (oid[2] << 8) | oid[3];
+		memcpy(file->name, path, pathlen);
+		file->namelen = pathlen;
```

Comment:
why do you need this change?

Reply:
I was testing and forget to delete this, it works without it.